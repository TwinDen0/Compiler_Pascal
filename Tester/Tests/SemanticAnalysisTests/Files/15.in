procedure a (x,y : integer);
begin

end;
begin
a(5);
end.