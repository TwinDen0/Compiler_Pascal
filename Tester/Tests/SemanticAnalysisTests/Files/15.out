(1, 6) Wrong number of parameters specified for call to "a"